crates/block-producer/src/withdrawal.rs

```diff
@@ -37,7 +37,7 @@ pub fn generate(
     contracts_dep: &ContractsCellDep,
     withdrawal_extras: &HashMap<H256, WithdrawalRequestExtra>,
 ) -> Result<Option<GeneratedWithdrawals>> {
-    if block.withdrawals().is_empty() && finalized_custodians.cells_info.is_empty() {
+    if block.withdrawals().is_empty() && finalized_custodians.cells_info.len() <= 1 {
```

Comment:
We can still generate withdrawal if there is only one finalize cell, why return none?

Reply:
Yes, but it's useless. To merge finalized custodian cells, we need at least two cells. We can safely return none here.